testing/tests/DevExpress.ui.widgets/responsiveBox.tests.js

```diff
@@ -301,6 +301,48 @@ QUnit.module('layouting', moduleConfig, () => {
         assert.equal($second.height(), maxSize, 'height is max-height');
     });
 
+    QUnit.test('default value of minWidth and minHeight in an item for a row direction of the container. Flex strategy', function(assert) {
+        const $responsiveBox = $('#responsiveBox').dxResponsiveBox({
+            rows: [{ ratio: 1 }],
+            cols: [{ ratio: 1 }, { ratio: 1 }],
+            items: [
+                { location: { row: 0, col: 0 }, html: '<div class=\'first\'></div>' },
+                { location: { row: 0, col: 1 }, html: '<div class=\'second\'></div>' }
+            ],
+            _layoutStrategy: 'flex'
+        });
+
+        const $first = $responsiveBox.find('.first').closest('.dx-box-item');
+        const $second = $responsiveBox.find('.second').closest('.dx-box-item');
+        const getStyleValue = ($element, styleName) => $element.get(0).style[styleName];
+
+        assert.equal(getStyleValue($first, 'minWidth'), '0px', 'minWidth of the first item');
```

Comment:
I would prefer to check here the resulting coordinates using the getBoundingClientRect for two fixed sizes of the parent element.
The 'minSize: auto/miSize: 0' are source settings for 'layouting' code, they can lead to different results in IE/Chrome/Edge/ios and we can misunderstand these values as 'incorrect' later.
For T878630 scenario getBoundingClientRect allows to easily create a stable and fast 'screenshot' test (usually 'width/height < 100px' requires '< 1px' rounding)
Something like [FormLayoutTestWrapper.checkElementPosition](https://github.com/DevExpress/DevExtreme/blob/21c5577ddc46de78f287cd6abfa8467b6a28a61e/testing/helpers/FormLayoutTestWrapper.js#L29)